dubbo-config/dubbo-config-spring/src/main/java/com/alibaba/dubbo/config/spring/schema/DubboBeanDefinitionParser.java

```diff
@@ -202,7 +202,13 @@ private static BeanDefinition parse(Element element, ParserContext parserContext
                                     String throwMethod = value.substring(index + 1);
                                     reference = new RuntimeBeanReference(throwRef);
                                     beanDefinition.getPropertyValues().addPropertyValue("onthrowMethod", throwMethod);
-                                } else {
+                                } else if ("oninvoke".equals(property)) {
+                                    int index = value.lastIndexOf(".");
+                                    String throwRef = value.substring(0, index);
```

Comment:
I think the variable name `throwRef` should change to `invokeRef`.

Reply:
I have fix the variable  name